routes/api/contacts.js

```diff
@@ -45,9 +40,11 @@ router.get("/:contactId", async (req, res) => {
 });
 
 router.post("/", async (req, res) => {
+  const body = req.body;
+  const contact = new Contacts(body);
+  const validate = validateJoi(contact);
```

Comment:
Validacja zawsze pierwsza, jeśli coś nie tak to kończysz i zwracasz jakiś błąd. Potem idzie reszta operacji.